routes/configOverview.js

```diff
@@ -1,62 +1,24 @@
-
-var linz = require('../'),
-    async = require('async');
+var linz = require('../');
 
 /* GET /admin/config/:config/overview */
 var route = function (req, res) {
 
-	var locals = {
-		config: req.linz.config,
-		record: req.linz.record,
-		permissions: req.linz.config.linz.formtools.permissions,
-		formtools: req.linz.config.linz.formtools
-	};
-
-	async.series([
-
-        function (cb) {
-
-            req.linz.config.linz.formtools.overview.summary.renderer(req.linz.record, req.linz.config, function (err, content) {
-
-                if (err) {
-                    return cb(err);
-                }
-
-                locals.overviewSummary = content;
-
-                return cb(null);
-
-            });
-
-        },
-
-		function (cb) {
-
-			if (!req.linz.config.linz.formtools.overview.body) {
-
-				return cb(null);
-
-			}
-
-			req.linz.config.linz.formtools.overview.body(req.linz.record, req.linz.config, function (err, content) {
-
-                if (err) {
-                    return cb(err);
-                }
-
-				locals.overviewBody = content;
-
-				return cb(null);
-
-			});
+    var locals = {
+        config: req.linz.config,
+        record: req.linz.record,
+        permissions: req.linz.config.linz.formtools.permissions,
+        formtools: req.linz.config.linz.formtools,
+        overview: req.linz.overview
+    };
 
-		}
+    if (Array.isArray(locals.overview.body)) {
 
-	], function (err, results) {
+        // Set tabId to each tab in locals.overview.body
+        linz.formtools.overview.setTabId(locals.overview.body);
 
-		res.render(linz.api.views.viewPath('configOverview.jade'), locals);
+    }
 
-	});
+    res.render(linz.api.views.viewPath('configOverview.jade'), locals);
```

Comment:
Can you please add `return` at the beginning of this line of code to follow our standard practice?